client/my-sites/upgrades/domain-management/email-forwarding/email-forwarding-item.jsx

```diff
@@ -29,9 +29,41 @@ const EmailForwardingItem = React.createClass( {
 			this.recordEvent( 'deleteClick', domain, mailbox, forward_address, ! Boolean( error ) );
 
 			if ( error ) {
-				notices.error( error.message );
+				notices.error( error.message || this.translate( 'Failed to delete email forwarding record. Please try again or contact customer support if error persists.' ) );
 			} else {
-				this.props.successNotice( this.translate( 'Yay, %(email)s has been successfully deleted!', { args: { email: email } } ) );
+				notices.success(
+					this.translate( 'Yay, %(email)s has been successfully deleted!', {
+						args: {
+							email: email
+						}
+					} ), {
+						duration: 5000
+					} );
+			}
+		} );
+	},
+
+	resendVerificationEmail: function() {
+		const { temporary, domain, mailbox, forward_address, email } = this.props.emailData;
+
+		if ( temporary ) {
+			return;
+		}
+
+		upgradesActions.resendVerificationEmailForwarding( domain, mailbox, ( error, response ) => {
+			this.recordEvent( 'resendVerificationClick', domain, mailbox, forward_address, ! Boolean( error ) );
```

Comment:
Actually, I think just `! error` is the right approach here.
`!! error` is a good hack for treating any value as a "truthy" (non-negated) boolean value. That is: negate it first to treat it as a boolean, and then negate it again to get the "original" boolean value. So, no negation in the end.
Since we have a negation here, just `! error` will make sure that `error` is treated as a boolean value (and then negated).